apps/agent/.env.example

```diff
@@ -1,5 +1,15 @@
 
 TINYBIRD_TOKEN=
-DATABASE_DSN=
+
+DATABASE_DSN=USER:PASSWORD@tcp(HOST)/?tls=true
+
+# a random token which should be simillar to the value used in /apps/web/.env
```

Comment:
*a random token that should be the same as in /apps/web/.env

Reply:
`openssl rand -base64 32` should work on linux/mac, not sure about windows